src/modules/bruteforce.rs

```diff
@@ -21,25 +21,30 @@ impl Module for Bruteforce {
     }
 
     fn signature(&self) -> String {
-        "./bruteforce.sh".to_string()
+        "./bruteforce.sh ./hashes.txt".to_string()
     }
 
     async fn run(&self, app_config: &AppConfig) {
         let mut rng = rand::thread_rng();
-        let password = &format!(
-            "{}{:02}",
-            FirstName(EN).fake::<&str>().to_lowercase(),
-            rng.gen_range(0..99)
-        );
-        let hash_str: &str = &sha256(password);
-
-        print(format!("SHA256 value: {hash_str}",)).await;
+
+        let n_parallel = rng.gen_range(2..5);
```

Comment:
```suggestion
        let n_parallel = rng.gen_range(2..10);
```
Let's allow it to go a little crazy here.